src/coreComponents/physicsSolvers/fluidFlow/CompositionalMultiphaseBase.cpp

```diff
@@ -1086,11 +1095,19 @@ void CompositionalMultiphaseBase::applyDirichletBC( real64 const time,
                    "ElementRegions",
                    viewKeyStruct::pressureString(),
                    [&]( FieldSpecificationBase const & fs,
-                        string const &,
+                        string const & setName,
                         SortedArrayView< localIndex const > const & targetSet,
                         Group & subRegion,
                         string const & )
   {
+    if( fs.getLogLevel() >= 2 && m_nonlinearSolverParameters.m_numNewtonIterations == 0 )
+    {
+      localIndex const numTargetElems = MpiWrapper::sum( targetSet.size() );
+      GEOSX_LOG_RANK_0( GEOSX_FMT(
+                          "CompositionalMultiphaseBase {}: at time {}s, the <{}> boundary condition named \"{}\" is applied to the element set named \"{}\" in subRegion \"{}\". \nThe total number of target elements (including ghost elements) is {}. \nNote that if this number is equal to zero for all subRegions, the boundary condition will not be applied on this element set.",
```

Comment:
Not that there is a definitive rule on GEOSX on this, but I've always preferred single quotes to refer to named entities in error messages, if not for any other reason, then at least so we don't have to keep escaping double quotes in format strings. I also suggested removing "named" because the following quotes already signal that it's referring to a name (at least in my opinion).
```suggestion
                          "CompositionalMultiphaseBase {}: at time {}s, the <{}> boundary condition '{}' is applied to the element set '{}' in subRegion '{}'. \nThe total number of target elements (including ghost elements) is {}. \nNote that if this number is equal to zero for all subRegions, the boundary condition will not be applied on this element set.",
```

Reply:
Done, I made the modifications here and propagated it across the solver